src/pdm4ar/exercises_def/structures_time.py

```diff
@@ -15,6 +15,9 @@ def function_runner(*args, **kwargs):
 
     send_end = kwargs.pop("__send_end")
     function = kwargs.pop("__function")
+    ########
+    set_memory_limit(.3)
```

Comment:
Probably we should not set the limit based on a percentage of the free memory. But as an absolute value e.g. 8GB, so that it's fair among submissions.

Reply:
a fixed value might be dangerous when people are running on their local machines.... (e.g. if a student has only 6gb ram setting 8 would fail).
I think that correct solutions also should be so far from the limit that we can have "unfairness" given by the variability of the current free memory